packages/designmanual/stories/pages/SubjectPage.jsx

```diff
@@ -515,13 +504,23 @@ const SubjectPage = ({
           </LayoutItem>
         </OneColumn>
       </div>
+      {/* Stored for later. 
       {bannerBackground && <SubjectBanner image={bannerBackground} negativeTopMargin={moveBannerUp} />}
       {subjectAboutHeading && <OneColumn wide>{subjectAbout(subjectAboutHeading, subjectAboutDescription)}</OneColumn>}
       {subjectContentCards && (
         <SubjectCarousel wideScreen subjects={subjectContentCards} title="Litt forskjellig fra faget" subjectPage />
-      )}
+      )} */}
+
       <OneColumn wide>
-        <Breadcrumblist isVisible={showBreadCrumb} items={breadcrumbItems} onNav={handleNav} />
+        {message && (
+          <Breadcrumblist
+            isVisible={showBreadCrumb}
+            items={breadcrumbItems}
+            onNav={handleNav}
+            messageBoxMini={messageBoxMini}
+          />
+        )}
+        {!message && <Breadcrumblist isVisible={showBreadCrumb} items={breadcrumbItems} onNav={handleNav} />}
```

Comment:
Om du sender inn messageBoxMini så vil jo den kun rendres dersom den er truthy.
```suggestion
    <Breadcrumblist
        isVisible={showBreadCrumb}
        items={breadcrumbItems}
        onNav={handleNav}
        messageBoxMini={messageBoxMini}
    />
```